lib/src/main/java/growthbook/sdk/java/FeatureEvaluator.java

```diff
@@ -374,13 +382,14 @@ public <ValueType> FeatureResult<ValueType> evaluateFeature(
                     .value(value)
                     .build();
 
-            if (featureUsageCallback != null) {
-                featureUsageCallback.onFeatureUsage(key, defaultValueFeatureResult);
+            if (featureUsageCallbackWithUser != null) {
+                featureUsageCallbackWithUser.onFeatureUsage(key, defaultValueFeatureResult, context.getUser());
             }
 
             // Return (value = defaultValue or null, source = defaultValue)
             return defaultValueFeatureResult;
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Is this an intentional change?

Reply:
yeah, a debug statement. I'll remove it!